src/service/viewport/viewport-binding-ios-embed-sd.js

```diff
@@ -0,0 +1,440 @@
+/**
+ * Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Observable} from '../../observable';
+import {Services} from '../../services';
+import {ViewportBindingDef} from './viewport-binding-def';
+import {computedStyle, px, setImportantStyles} from '../../style';
+import {dev} from '../../log';
+import {htmlFor} from '../../static-template';
+import {isExperimentOn} from '../../experiments';
+import {layoutRectLtwh} from '../../layout-rect';
+import {waitForBody} from '../../dom';
+import {whenDocumentReady} from '../../document-ready';
+
+const TAG_ = 'Viewport';
+
+/**
+ * This includes only styles that are necessary for layout, but not normally
+ * inherited. The inherited styles, such as `font`, are inherited automatically.
+ * The main styles that need to be additionally handled are `display`,
+ * `padding`, etc.
+ */
+const INHERIT_STYLES = [
```

Comment:
Out of curiosity, how did you get this list of styles?

Reply:
I took a list of all styles with default value _not_ `inherited` and then removed dupes and some styles that didn't make sense to me.